packages/react/.storybook/manager.js

```diff
@@ -0,0 +1,9 @@
+import theme from './theme';
+import { addons } from '@storybook/addons';
+
+addons.setConfig({
+  theme: theme,
+  sidebar: {
+    showRoots: true,
+  },
+});
```

Comment:
this file is added b/c of deprecations that were put in place in 6.0. showRoots/theme are now moved into this manager file and set through setConfig rather than as params in the preview file.